pom.xml

```diff
@@ -2,23 +2,30 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
     <groupId>com.senla</groupId>
     <artifactId>bookstore</artifactId>
-    <packaging>jar</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <version>1.0</version>
+    <modules>
+        <module>database</module>
+        <module>entity</module>
+        <module>services</module>
+        <module>ui</module>
+        <module>DI</module>
+        <module>json</module>
+        <module>Test</module>
```

Comment:
you should have only 3 modules: 
1 DI
2 UI(your menu and data representation layer)
3 core module(core business logic)

Reply:
fixed